perception/image_projection_based_fusion/src/utils/geometry.cpp

```diff
@@ -122,7 +122,7 @@ void boundingBoxToVertices(
 
   const auto position = Eigen::Vector3d(pose.position.x, pose.position.y, pose.position.z);
   const auto orientation = Eigen::Quaterniond(
-    pose.orientation.w, pose.orientation.x, pose.orientation.y, pose.orientation.z);
+    pose.orientation.x, pose.orientation.y, pose.orientation.z, pose.orientation.w);
```

Comment:
Is it correct?
https://eigen.tuxfamily.org/dox/classEigen_1_1Quaternion.html

Reply:
fixed by retrieve new commit